L1Trigger/TrackFindingTracklet/interface/CircularBuffer.h

```diff
@@ -0,0 +1,63 @@
+#ifndef CIRCULARBUFFER_H
+#define CIRCULARBUFFER_H
+
+#include <assert.h>
+#include <vector>
+
+using namespace std;
+
+template <class T>
+class CircularBuffer {
```

Comment:
would it make sense to use [`boost::circular_buffer`](https://www.boost.org/doc/libs/1_72_0/doc/html/circular_buffer.html) rather than implementing your own ?